local_pathfinding/node_navigate.py

```diff
@@ -134,17 +131,35 @@ def get_desired_heading(self) -> float:
             float: The desired heading if all subscribers are active, else a number that violates
                 the heading convention.
         """
+        _, planner = self.get_params()
```

Comment:
Should go right before `update_if_needed()` (want to check if all subs are active before doing anything)